user/trusty-ci-environment.md

```diff
@@ -0,0 +1,278 @@
+### Networking tools
+
+curl, wget, OpenSSL, rsync
+
+### Docker
+
+Docker 1.8.2 is installed. We use the official [Docker apt
+repository](https://blog.docker.com/2015/07/new-apt-and-yum-repos/), so
+you can easily install another version with `apt-get` if needed.
+
+[docker-compose](https://docs.docker.com/compose/)
+version [1.4.2](https://github.com/docker/compose/releases/tag/1.4.2) is also
+installed.
+
+See our [Using Docker in Builds](/user/docker/)
+section for more details on our Docker beta and its usage.
+
+### Runtimes
+
+#### Ruby
+
+[rvm](https://rvm.io/rvm/about) is installed and we've pre-installed the following Ruby versions
+with it.
+
+
+- jruby-9.0.1.0 [ x86_64 ]
+- ruby-1.9.3-p551 [ x86_64 ]
+- ruby-2.0.0-p647 [ x86_64 ]
+- ruby-2.1.7 [ x86_64 ]
+- ruby-2.2.3 [ x86_64 ]
+
+#### Python
+
+We pre-install the following python versions
+
+- pypy 2.6.1
+- pypy3 2.4.0
+- python 2.6.9
+- python 2.7.10
+- python 3.2.6
+- python 3.3.6
+- python 3.4.3
+- python 3.5.0
+
+[pyenv](https://github.com/yyuu/pyenv#simple-python-version-management-pyenv) is also installed.
+
+#### Node.JS
+
+[nvm](https://github.com/creationix/nvm#installation) is installed and we've pre-installed the following Node versions
+with it
+
+- iojs-v1.6.4
+- v0.6.21
+- v0.8.28
+- v0.10.40
+- v0.11.16
+- v0.12.7
+- 4.1.2
+
+#### Go
+
+[gimme](https://github.com/travis-ci/gimme#gimme) is installed and we've pre-installed the following Go versions
+with it
+
+- go 1.0.3
+- go 1.1.2
+- go 1.2.2
+- go 1.3.3
+- go 1.4.2
+- go 1.5.1
+
+#### JVM
+
+- We install the latest versions OpenJDK versions from the official Ubuntu
+  Trusty packages. (1.8.0_60 as of 2015-10-13)
+- We install the latest Oracle JDK versions from Oracle.
+- [jdk_switcher](https://github.com/michaelklishin/jdk_switcher#what-jdk-switcher-is) is installed if you need another version.
+- gradle 2.5 is installed.
+- Maven 3.1.1 is installed.
+- leiningen 2.5.1 is installed.
+- sbt 0.13.9
+
+#### PHP
+
+We do not currently have any PHP versions pre-installed in the **mega**
+image. This is at the top of our list for our next update.
+
+
+But we do have [phpenv](https://github.com/phpenv/phpenv) installed and
+it's able to install the following PHP versions.
+
+- PHP 5.3
+- PHP 5.4
+- PHP 5.5
+- PHP 5.6
+- PHP 7.0.0 RCs
+
+_Note: We're unable to build **PHP 5.2** on Trusty so far, so trying to use
+it will result in a build failure when phpenv fails to compile it_
+
+#### Other runtimes
+
+Other runtimes are available to install using `apt-get` and the official
+Ubuntu packages, any third party PPAs or custom scripts that you
+manually setup in your `.travis.yml`
+
+### Data Stores
+
+We pre-install the following services and they can be activated with the built-in [services](/user/database-setup/) support.
+
+* PostgreSQL 9.4.5
+* SQLite 3.8.2
+* Redis 3.0.4
+
+*Note: The following services that are included with our Precise images
+are **not included** in the Trusty **mega** images at this time but will
```

Comment:
I'd say "may be added in the coming months".  I'm not planning to port over Riak & Neo4J, for example.